README.md

```diff
@@ -51,6 +51,39 @@ to show the container's journal information on the host, using journalctl.
 journalctl -M CONTAINER_UUID
 ```
 
+To use this directly with runc, modify or add the following to config.json.
+
+```
+    "hooks": {
+        "prestart": [
+            {
+                "path": "/usr/libexec/oci/hooks.d/oci-systemd-hook"
+            }
+        ]
+    },
+...
+    "process": {
+        "capabilities": {
+...
+                "CAP_AUDIT_WRITE",
+                "CAP_KILL",
+                "CAP_NET_BIND_SERVICE",
+                "CAP_MKNOD",
+                "CAP_CHOWN",
+                "CAP_DAC_OVERRIDE",
+                "CAP_FSETID",
+                "CAP_FOWNER",
+                "CAP_NET_RAW",
+                "CAP_SETGID",
+                "CAP_SETUID",
+                "CAP_SETFCAP",
+                "CAP_SETPCAP",
+                "CAP_SYS_CHROOT"
```

Comment:
nit time, but should there be a comma at the end of this line?

Reply:
Well, no need comma if it is the end of the list.

Reply:
Right, but with the '...' that followed, it led me to believe there could be more in the list after this.

Reply:
But I'm fine as it is, it's a nit and I suspect most folks will add the comma if necessary.

Reply:
I meant that '...'  indicate there are other sections other than capabilities in config.json. That capabilities list is sufficient to run systemd. If I add a comma there, it may lead to people believe that more capabilities are required which is not the case.